imap_processing/idex/idex_l2b.py

```diff
@@ -120,3 +122,55 @@ def round_spin_phases(spin_phases: xr.DataArray) -> xr.DataArray:
     # Calculate nearest quadrant value.
     # Use mod to wrap values > 315 to 0.
     return (quadrant_size * (shifted_spin_phases / quadrant_size).astype(int)) % 360
+
+
+def get_science_acquire_timestamps(
```

Comment:
This is the only place you use "acquire" so maybe change it to "acquisition" for consistency with everywhere else?
```suggestion
def get_science_acquisition_timestamps(
```